src/python/ensembl/io/genomio/gff3/process_gff3.py

```diff
@@ -414,6 +448,9 @@ def _normalize_transcripts(self, gene: SeqFeature, fail_types) -> SeqFeature:
             transcript_number = count + 1
             transcript.id = self.normalize_transcript_id(gene.id, transcript_number)
 
+            if transcript.type in ("C_gene_segment", "V_gene_segment"):
```

Comment:
This check is done twice: here, and in the format_gene_segments. Might as well keep only the one in the method, but that'll require removing the "else" condition in it.